src/App.js

```diff
@@ -1,28 +1,32 @@
 import React from 'react';
 import Todo from './components/TodoComponents/Todo.js';
+import TodoList from './components/TodoComponents/TodoList.js';
 import TodoForm from './components/TodoComponents/TodoForm.js';
 
+
+
 class App extends React.Component {
   constructor(){
     super();
     this.state = {
-      task: ''
+      tasks: [{task: 'do homework'},{task: 'complete MVP'}]
```

Comment:
Nice work on your progress so far Francis! Try to indent the objects in this section to make things more readable. Other than that, I'll leave you to work. :) 👍 👍